tests/rbr/adjustRegionWithOwnerTest.java

```diff
@@ -0,0 +1,36 @@
+package rbr;
+
+import java.util.*;
+import org.junit.*;
+import util.*;
+
+public class adjustRegionWithOwnerTest {
+    Set<Character> ip = new HashSet<>();
+    Set<Character> op = new HashSet<>();
+    Set<Vertex> dst = new HashSet<>();
+    private RBR rbr;
+
+    @Before
+    public void setUp() {
+        Graph graph = RegularGraphBuilder.generateGraph(4,4);
+        rbr = new RBR(graph);
+        ip.add('I');
+        op.add('N');
+        op.add('E');
+        dst.add(new Vertex("0.3"));
+        dst.add(new Vertex("1.3"));
+        dst.add(new Vertex("1.2"));
+    }
+
+    @Test
+    public void adjustedRegionsOwnerInsideRegion() throws Exception {
```

Comment:
I'd add a test showing that the adjustedRegion is the same original region when the owner is not inside the box.